edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -371,6 +332,34 @@ def _is_jwt_cookie_and_session_user_mismatch(self, request, jwt_user_id):
 
         return True
 
+    def _get_and_monitor_jwt_cookie_lms_user_id(self, request):
+        """
+        Returns the LMS user id from the JWT cookie, or None if not found
+
+        Notes:
+        - Must only be called in the case of a JWT cookie.
```

Comment:
This sentence used to end with "failure", but the removal makes it unclear to me. Maybe this?

```suggestion
        - Must only be called when a JWT cookie is present.
```

Reply:
I deleted the line instead. Thanks.